src/Azure/Orleans.Persistence.AzureStorage/Storage/StorageExceptionExtensions.cs

```diff
@@ -6,19 +6,9 @@ namespace Orleans.Storage
 {
     internal static class StorageExceptionExtensions
```

Comment:
Just in case you weren't aware - some of these methods may benefit from [TableErrorCode](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/tables/Azure.Data.Tables/src/TableErrorCode.cs#L10)

ex:  https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/tables/Azure.Data.Tables/src/TableServiceClient.cs#L660